### DISCUSSION TITLE
可用性告警与监控状态关系

### DISCUSSION THREAD

**Discussion by hxJava666** [apache/hertzbeat#2774] [Q&A]
可用性告警没有发出告警，导致监控状态一直没有改变，但是当前这个监控任务并没有去跑任务，状态应该是不可用才对。

**Comment by tomsun28** [comment]
Hi, The current availability alert affects the monitoring status, but we are refactoring this alert part and it will not be affected later.

**Reply by liyfbeat** [reply]
Well, what is the current progress? It seems that version 1.7.1 still has this problem.
